pkg/analyze/host_time.go

```diff
@@ -38,7 +38,7 @@ func (a *AnalyzeHostTime) Analyze(
 		getCollectedFileContents,
 		collect.HostTimePath,
 		collect.NodeInfoBaseDir,
-		collect.HostMemoryFileName,
+		collect.HostTimeFileName,
```

Comment:
Noticed the time collector was using the memory collector file name, not really part of the bulk of this changes but thought it would be a useful fix.